sys-boot/syslinux/syslinux-6.04_pre1-r2.ebuild

```diff
@@ -74,8 +74,8 @@ src_prepare() {
 			"
 	fi
 	case ${ARCH} in
-		amd64)	loaderarch="efi64" ;;
-		x86)	loaderarch="efi32" ;;
+		amd64)	loaderarch=$(usex bios bios efi64) ;;
```

Comment:
should the efi32/64 have been outside the `)`?

Reply:
By default only efi32/64 is provided, if bios flag is enabled then only bios should be supported -  so that was intentional. I did an update of metadata.xml to get rid of inconsistent indentation error.

Reply:
> By default only efi32/64 is provided

I don't actually think that's true: the bios code is being compiled in src_install by accident. This PR fails to address that.